cmd/token_test.go

```diff
@@ -89,6 +89,46 @@ func TestTokenWithGCEAK(t *testing.T) {
 	}
 }
 
+func TestCustomEventLogFile(t *testing.T) {
+	if _, err := os.Stat("/dev/tpm0"); os.IsNotExist(err) {
+		t.Skip("Skipping test: /dev/tpm0 not found")
+	}
+
+	ExternalTPM = nil
```

Comment:
You don't need to set ExternalTPM; it defaults to nil.

Reply:
Without this assignment, we would have a concurrency error. See commit e5558fc74b06f943ce723d87f7884c990e584eeb vs commit 0f65c4c6b6250735d7cfdbdac506bca5478ec666